decidim-dev/lib/decidim/dev/test/rspec_support/feature.rb

```diff
@@ -35,6 +35,7 @@ class DummyResourcesController < ActionController::Base
     def show
       @commentable = DummyResource.find(params[:id])
       render inline: %{
+        <%= csrf_meta_tags %>
```

Comment:
Because the csrf token is used when reporting a comment. It doesn't use the GraphQL API